yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -54,8 +54,8 @@ type UnsupportedFeature struct {
 
 var assessMigrationCmd = &cobra.Command{
 	Use:   "assess-migration",
-	Short: "Assess the migration from source database to YugabyteDB.",
-	Long:  `Assess the migration from source database to YugabyteDB.`,
+	Short: "Assess the migration from PostgreSQL source database to YugabyteDB.",
```

Comment:
nit: let's still keep it `source` , but we can frame it as `source (PostgreSQL) database ...` ? 
That way, it would be indicative, that we plan to support other database types in the future.

Reply:
good point.